test/featuregates_linter/test_data/versioned_feature_list.yaml

```diff
@@ -856,6 +856,12 @@
     lockToDefault: false
     preRelease: Beta
     version: "1.24"
+- name: OrderedNamespaceDeletion
+  versionedSpecs:
+  - default: false
+    lockToDefault: false
+    preRelease: Beta
```

Comment:
the release note does not match this, right? which one are we going to use alpha or beta?